wp-content/themes/sample-theme/blocks/src/core--button/css/editor.scss

```diff
@@ -0,0 +1,10 @@
+@use 'sass:meta';
+
+.editor-styles-wrapper {
+	// Import all default styles.
+	// @include meta.load-css( 'style.scss' );
```

Comment:
Here we could import all theme-level styles that impact the behaviour of this block and have them be namespaced under the `.editor-style-wrapper` wrapper.